mail_send_restriction/views/mail_message_subtype_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+   <record model="ir.ui.view" id="view_mail_message_subtype_form">
+        <field name="name">mail.message.subtype.form</field>
+        <field name="model">mail.message.subtype</field>
+        <field name="inherit_id" ref="mail.view_mail_message_subtype_form" />
+        <field name="arch" type="xml">
+            <field name="res_model" position="after">
+                <field name="allow_send_model_ids" widget="many2many_tags" />
```

Comment:
Please make the field invisible when it's not relevant.  We may add a technical boolean field (hide_allow_send_model) in `mail.message.subtype` to control the behavior as if we only want to show the field for `mail.mt_comment`.